docker/src/main/java/clocker/docker/location/DockerContainerLocation.java

```diff
@@ -188,12 +190,13 @@ public int execScript(Map<String,?> props, String summaryForLogging, List<String
         for (String commandString : filtered) {
             parseDockerCallback(commandString);
         }
-        if (getOwner().config().get(DockerContainer.DOCKER_USE_SSH)) {
+        boolean entitySsh = Boolean.TRUE.equals(entity.config().get(DockerContainer.DOCKER_USE_SSH));
+        boolean dockerSsh = Boolean.TRUE.equals(getOwner().config().get(DockerContainer.DOCKER_USE_SSH));
+        if (entitySsh && dockerSsh) {
```

Comment:
The default value of that config key is `Boolean.TRUE` so we are _really_ checking if either of the application or the Clocker blueprints have set it to `false`... Maybe a comment to that effect would be useful?